docs/governance/finos-poc.md

```diff
@@ -0,0 +1,16 @@
+# FINOS Point-of-Contact (POC)
```

Comment:
In some places we link to the FINOS POC section here: https://github.com/finos/common-cloud-controls/blob/main/docs/governance/steering/charter.md#finos-point-of-contact

Should these links be updated to point to this page?

Reply:
Added